spec/views/user_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe 'User view', type: :system do
+  before(:all) do
+    lilly = User.create(name: 'Lilly',
+                        photo: 'https://randomuser.me/api/portraits/women/40.jpg',
+                        bio: 'Teacher from Poland.')
+    User.create(name: 'Tom',
+                photo: 'https://randomuser.me/api/portraits/men/92.jpg',
+                bio: 'Teacher from Mexico.')
+    Post.create(author: lilly, title: 'new post', text: 'I love posting')
+    Post.create(author: lilly, title: 'second post', text: 'yeahhh')
+    Post.create(author: lilly, title: 'third post', text: 'uuuu')
+    Post.create(author: lilly, title: 'fourth post', text: 'eee')
+  end
+
+  describe 'index page' do
+    it 'shows the users names' do
+      visit users_path
+      expect(page).to have_content('Tom')
+      expect(page).to have_content('Lilly')
+    end
+
+    it 'shows the users profile picture' do
+      visit users_path
+      images = page.all('img')
+      expect(images[0]['src']).to have_content('https://randomuser.me/api/portraits/men/92.jpg')
+      expect(images[1]['src']).to have_content('https://randomuser.me/api/portraits/women/40.jpg')
+    end
+
+    it 'shows the number of posts each user has written' do
+      visit users_path
+      posts_counter = page.all('.posts-counter')
+      expect(posts_counter[0]).to have_content('Number of posts: 0')
+      expect(posts_counter[1]).to have_content('Number of posts: 4')
+    end
+
+    it 'when I click on a user, I am redirected to that users show page' do
```

Comment:
- [ ] Just like the previous reviewer opined, these 3 tests are failing. Kindly see the attached image below try to fix them 👍 
![image](https://user-images.githubusercontent.com/46839548/200654014-cb37b9ab-45ea-4643-95de-9673e4e20242.png)